redfish-core/lib/log_services.hpp

```diff
@@ -1856,80 +2150,208 @@ inline void requestRoutesDBusEventLogEntry(App& app)
             [](const crow::Request& req,
                const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
                const std::string& entryId) {
-                std::optional<bool> resolved;
+                auto getHiddenProperty = [req, asyncResp,
+                                          entryId](bool hiddenPropVal) {
+                    if (hiddenPropVal)
+                    {
+                        messages::resourceNotFound(asyncResp->res, "LogEntry",
+                                                   entryId);
+                        return;
+                    }
+                    updateResolvedProperty(req, asyncResp, entryId);
+                };
+                getHiddenPropertyValue(asyncResp, entryId,
+                                       std::move(getHiddenProperty));
+            });
 
-                if (!json_util::readJson(req, asyncResp->res, "Resolved",
-                                         resolved))
-                {
-                    return;
-                }
-                BMCWEB_LOG_DEBUG << "Set Resolved";
+    BMCWEB_ROUTE(
+        app, "/redfish/v1/Systems/system/LogServices/EventLog/Entries/<str>/")
+        .privileges(redfish::privileges::deleteLogEntry)
+        .methods(boost::beast::http::verb::delete_)(
+            [](const crow::Request&,
+               const std::shared_ptr<bmcweb::AsyncResp>& asyncResp,
+               const std::string& param) {
+                BMCWEB_LOG_DEBUG << "Do delete single event entries.";
+                std::string entryID = param;
+                dbus::utility::escapePathForDbus(entryID);
+
+                auto getHiddenProperty = [asyncResp,
```

Comment:
This lambda doesn't really get the hidden property, rather it deletes an event log.

Reply:
Done